Thrift.g4

```diff
@@ -186,7 +186,7 @@ TYPE_STRING: 'string';
 TYPE_BINARY: 'binary';
 
 LITERAL_VALUE
-    : (('"' ~'"'* '"') | ('\'' ~'\''* '\''))
+    : (( '"' (('\\' ('n'|'t'|'"'|'\''|'\\') ) | ~('\\' | '"'))* '"') | ('\'' ~'\''* '\''))
```

Comment:
the right half part need to complete `('\'' ~'\''* '\'')`

Reply:
origin:  `('"' ~'"'* '"')`  the middle assume not contain any `"`

now: `( '"' (('\\' ('n'|'t'|'"'|'\''|'\\') ) | ~('\\' | '"'))* '"')`

the  middle will contain `\n` or `\t` or `\\` or `\"` or `\'`  or (any character that not was `\\` or `"` )